README.md

```diff
@@ -144,7 +148,9 @@ Once you've created your Camel K task, you can use auto-complete to explore the
 }
 ```
 
-When you're done, save the file and call command from palette `Tasks: Run Task`. You will see the command listed with the label that you provided previously.
+When you're done, you have two options to use it:
```

Comment:
Might change this to:

When you've defined your Task, you can use one of two options to execute it: